client/kafka/v2/kafka.go

```diff
@@ -72,9 +72,14 @@ func New(brokers []string) *Builder {
 		ee = append(ee, errors.New("brokers are empty or have an empty value"))
 	}
 
+	cfg := sarama.NewConfig()
+	cfg.Net.MaxOpenRequests = 1
```

Comment:
Hey, could this be a performance impacting change? Given that there might be services out there with a lot of messages to process.

Reply:
It would make the producers use only 1 open request, so if some service is producing a lot of messages with the same producer client those would not go slower, I guess.

But can those services trade the extra speed (of quantity unknown to me) for the loss of messages in case of broker problems? In such cases perhaps they should override the default configuration?

Reply:
Makes sense. Pls add also any necessary documentation to it because this might be a breaking behavior and the end user should know what they are bying.

Reply:
I have updated GoDoc and `Kafka.md`, however perhaps we miss some documentation for the v2 Kafka consumer? :thinking:

Reply:
LGTM